compiler/executable/match_/planner/mod.rs

```diff
@@ -30,11 +30,18 @@ use crate::{
     },
     ExecutorVariable, VariablePosition,
 };
+use crate::executable::match_::planner::plan::QueryPlanningError;
 
 pub mod match_executable;
 pub mod plan;
 pub(crate) mod vertex;
 
+typedb_error! {
+    pub MatchCompilationError(component = "Match compiler", prefix = "MCL") {
```

Comment:
Can you update the other error messages in `//compiler/executable` to use the `xxx Compiler` terminology? Also, I was using FCP (Fetch ComPiler), but if you think CL is a better suffix, please change those too to `FCL`